examples/bri-3/blockchain/blockchain.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, OnModuleInit } from '@nestjs/common';
+
+@Injectable()
+export class BlockchainService implements OnModuleInit {
+  async onModuleInit() {
+    // TODO: connect to blockchain
+  }
+
+  async deploy(contract: any) {
+    //TODO: deploy contract
+  }
+
+  async store(proof: string) {
```

Comment:
@biscuitdey if we are storing a content-addressable hash onchain what are we storing at the storage endpoint? the full witness, only the proof? where is that specified?

Reply:
@Therecanbeonlyone1969  Currently, we are not storing anything at the storage endpoint. But the code can be modified to store the proof at the storage endpoint, if required. As I am not using ZK circuits atm, there is no witness. Do we need to store the proof in our storage?